crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -412,8 +412,10 @@ where
             .mesh_key
             .intersects(MeshPipelineKey::NORMAL_PREPASS | MeshPipelineKey::DEFERRED_PREPASS)
         {
-            vertex_attributes.push(Mesh::ATTRIBUTE_NORMAL.at_shader_location(3));
-            shader_defs.push("NORMAL_PREPASS_OR_DEFERRED_PREPASS".into());
+            if layout.0.contains(Mesh::ATTRIBUTE_NORMAL) {
+                vertex_attributes.push(Mesh::ATTRIBUTE_NORMAL.at_shader_location(3));
+                shader_defs.push("NORMAL_PREPASS_OR_DEFERRED_PREPASS".into());
```

Comment:
I think this shader def shouldn't be behind that check because for a custom prepass you'd still want that shader def to be available.

Reply:
Ah, I missed that this is what controls whether or not Vertex and Vertex Output have a normal field. Yeah, you are correct, that should always be included.